strapi/strapi
# Added option to override the Base URL of the file upload

### What does it do?

Added an option to override the S3 file URL with custom URL provided by config. 

### Why is it needed?

If using a reverse proxy like Nginx or using a CDN like Cloudflare you maybe want to proxy all files threw your custom origin instead of using the AWS / S3 provided file location. 

### How to test it?

Configure plugin properly and upload a file via Strapi and check if the URL matches the correct url and file is delivered. 

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request
